spec/errors.md

```diff
@@ -278,28 +278,28 @@ for example when encountering a value with an unsupported type
 or an internally inconsistent set of options.
 
 > For example, attempting to format any of the following messages
-> might result in a _Formatting Error_ if done within a context that
->
-> 1. provides for the variable reference `$user` to resolve to
->    an object `{ name: 'Kat', id: 1234 }`,
-> 2. provides for the variable reference `$field` to resolve to
->    a string `'address'`, and
-> 3. uses a `:get` formatting function which requires its argument to be an object and
->    an option `field` to be provided with a string value,
+> might result in a _Formatting Error_ because the `operand` type doesn't
+> match the expectations of the formatting function:
 >
 > ```
-> Hello, {horse :get field=name}!
+> {{Hello, {horse :time} is not a date/time value!}}
 > ```
->
+> Attempting to format a non-numeric string as a number:
 > ```
-> Hello, {$user :get}!
+> .local $user = {|horse|}
+> {{Hello, {$user :number} is not a number!}}
 > ```
->
+> Attempting to format a number as a date:
 > ```
-> .local $id = {$user :get field=id}
-> {{Hello, {$id :get field=name}!}}
+> .local $num = {$arg :number}
+> {{Hello, {$num :date}!}}
 > ```
```

Comment:
We should consider numbers to be a valid data type from which to construct a date, and should not portray this as an error.

Reply:
Well........ sure, kind of. Are you expecting:
```
.local $num = {27}
{{{$num :datetime}}}
```
... to format as `01/01/1970 00:00:27`?

I could easily fix the example by reversing it:

```suggestion
.local $date = {$arg :date}
{{Hello, {$date :number}!}}
```

You don't expect dates to format as numbers, right?